incubator/sparkoperator/crds/crds.yaml

```
@@ -0,0 +1,5093 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
```

See above. Please add a label to both CRDs:

```suggestion
  labels:
    app.kubernetes.io/name: sparkoperator
```